go/client/cmd_paperprovision.go

```diff
@@ -23,16 +23,23 @@ func NewCmdPaperProvision(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cl
 		Action: func(c *cli.Context) {
 			cl.ChooseCommand(NewCmdPaperProvisionRunner(g), "paperprovision", c)
 		},
+		Flags: []cli.Flag{
+			cli.BoolFlag{
+				Name:  "keep-paper-key, k",
```

Comment:
why didn't you do it the other way; i.e., keep the same behavior by default, but then have `--discard-paper-key`?